lib/known_bugs.pm

```diff
@@ -59,4 +62,45 @@ sub create_list_of_serial_failures {
     return $serial_failures;
 }
 
+sub upload_journal {
+    my ($file) = @_;
+
+    my $failures = [];
```

Comment:
I guess that journal failures are to be set somewhere else like the serial failures right?

Reply:
Made it more explicit.